processing/src/main/java/io/druid/segment/IndexMergerV9.java

```diff
@@ -382,6 +383,15 @@ private void makeMetricsColumns(
                                          .build()
           );
           break;
+        case DOUBLE:
+          builder.setValueType(ValueType.DOUBLE);
+          builder.addSerde(
+              DoubleGenericColumnPartSerde.serializerBuilder()
```

Comment:
It uses different alignment than code above. Also I suggest to break the first method too:
```java
DoubleGenericColumnPartSerde
    .serializerBuilder()
    .withByteOrder(IndexIO.BYTE_ORDER)
    .withDelegate((DoubleColumnSerializer) writer)
    .build()
```